src/http/tracker_interface.rs

```diff
@@ -0,0 +1,100 @@
+use std::future::Future;
```

Comment:
Hi @WarmBeer should we also rename the mod `tracker_interface`?

Reply:
Yes, I forgot :)